sephora/game/objects/custom_ghost_role_spawners.dm

```diff
@@ -0,0 +1,56 @@
+/obj/effect/mob_spawn/human/sephora_syndicate_crew/

```

Comment:
Make these a subtype of `/obj/effect/mob_spawn/human/sephora/`, they don't need some awkward prefix which breaks with convention.

Reply:
Okay I've changed it.

It has created a spawner of the subtype above that just dumps a dead body on the ground, however.

Reply:
That's fine. It's not going to be used in proper gameplay and is merely a sorting tool.